pol-core/pol/module/uomod.cpp

```diff
@@ -4783,7 +4782,14 @@ bool UOExecutorModule::is_reserved_to_me( Item* item )
 
 BObjectImp* UOExecutorModule::mf_Shutdown()
 {
-  Clib::exit_signalled = true;
+  int exit_code;
+
+  if ( !getParam( 0, exit_code ) )
+  {
+    exit_code = 0;
+  }
```

Comment:
I was wrong.  The executor uses the argument count from the module function declaration in the .ecl file, so we don't need to bump the ecl version.

```suggestion
  int exit_code = 0;

  getParam( 0, exit_code );

```